.github/workflows/build-website.yml

```
@@ -51,16 +51,14 @@ jobs:
           ./ENV/bin/python ./tools/check-links http://localhost:9001/
           kill %1
         shell: bash
-      - name: check links (build)
+      - name: check links (apache2)
         run: |
-          cd website/www/
-          python3.7 -m http.server 9002 &
-          sleep 0.5  # HACK(strager): Wait for server to start.
-          cd ../../
-          ./ENV/bin/python ./tools/check-links http://localhost:9002/
-          kill %1
+          sudo DEBIAN_FRONTEND=noninteractive apt-get install apache2 rsync openssh-client openssh-server -y
+          sudo rsync --archive --delete website/www/ /var/www/html/
+          sudo sed -i -e '/None/ s/AllowOverride\ None/AllowOverride\ FileInfo/' /etc/apache2/apache2.conf
```

Nit: Is filtering to `None` really necessary? The substitution pattern already mentions `None`:
```suggestion
          sudo sed -i -e 's/AllowOverride\ None/AllowOverride\ FileInfo/' /etc/apache2/apache2.conf
```